### PR TITLE
Fix aggregation required rate

### PR DESCRIPTION
**Pull request by ghop02**
Turns out I wasn't passing the required reporting ratio through 🤦 so it was just aggregating all data regardless if it has missing data or not.  I'm running a snapshot to double check that these changes will turn out to reflect reality

### CODE REVIEW

File: test/libs/datasets/statistical_areas_test.py [hunk 1]
```diff
@@ -60,3 +64,29 @@ def test_aggregate():
         pd.to_datetime("2020-05-03"): 6,
         pd.to_datetime("2020-05-04"): 4,
     }
+
+
+@pytest.mark.parametrize("reporting_ratio,expected_na", [(0.8, False), (0.91, True)])
+def test_aggregate_reporting_ratio(reporting_ratio, expected_na):
+    ny_region = Region.from_state("NY")
+    az_region = Region.from_state("AZ")
+    us_region = Region.from_iso1("us")
```
Comment (TomGoBravo) [inline test/libs/datasets/statistical_areas_test.py:73]: unless there is good reason to use state -> US aggregation please use region types expected by CountyToCBSAAggregator.

**Review by TomGoBravo — APPROVED** [review]
I created https://trello.com/c/eXAtVkZO/807-expand-scope-of-lint-static-analysis-checks-to-catch-suspect-code

File: test/libs/datasets/statistical_areas_test.py (hunk 1 above)
Comment (ghop02) [reply test/libs/datasets/statistical_areas_test.py:73]: The best reason is for creating the test dataset, which expects regions

Comment (TomGoBravo) [reply test/libs/datasets/statistical_areas_test.py:73]: It is really just as easy to make county and CBSA regions: https://github.com/covid-projections/covid-data-model/blob/main/libs/pipeline.py#L108-L132

**Pull request merged**